web/Views/Home/ContactUs.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@model dynamic
+
+@{
+    @ViewData["Title"]
+}
+
+<div>
+    <h1>Contact Us</h1>
+    <br/>
+    <div>
+        <label> 
+        </label>
+        <a href="mailto:info@changeahead.org.uk"><i class="bi bi-envelope-fill"></i></a>
+        <a href="https://www.facebook.com/changeahead.org/"><i class="bi bi-facebook"></i></a>
+        <a href="https://twitter.com/changeaheaduk"><i class="bi bi-twitter"></i></a>
+        <a href="https://www.instagram.com/accounts/login/?next=/changeahead.org.uk/"><i class="bi bi-instagram"></i></a>
+        <a href="https://www.linkedin.com/company/changeaheaduk/"><i class="bi bi-linkedin"></i></a>
+        <a href="https://www.youtube.com/channel/UC0gZrrMIzBCfUMPnf4EOmzA"><i class="bi bi-youtube"></i></a>
+        <a href="https://www.tiktok.com/"><i class="bi bi-tiktok"></i></a>
+    </div>
+    <br/>
+    <strong>Please feel free to contact us if you have any queries:</strong>
+    <p>Email: info@changeahead.org.uk</p>
+    <p>Helpline: 03337 729 259 </p>
+    <p>Postal address: 85 Great Portland Street, First Floor, London, W1W 7LT</p>
+    <p>Or visit the Change Ahead website: <a href="https://www.changeahead.org.uk/">https://www.changeahead.org.uk/</a></p>
+    <br/>
+    @*TODO Populate this FAQ section with whatever questions and answers the client would like*@
+    <strong>Frequently Asked Questions</strong>
+    <p>[FAQS will be here, probably with some appropriate collapse feature]</p>
```

Comment:
Rather than this comment-esque text, I'd leave it empty and raise a ticket to create the expandable/collapsible FAQs section (with some lorem ipsum text in place of real questions and answers for the time being)

Reply:
I've raised a ticket for formatting on the Contact Us page, which would include figuring out what to do with the FAQ section.